addons/addon-base-raas/packages/serverless-packer/index.js

```diff
@@ -112,6 +118,16 @@ class ServerlessPackerPlugin {
     });
     return varArgs;
   }
+
+  async addVersionInfo(stageName) {
+    this.serverless.cli.log('Adding versionNumber and versionDate to stage file');
+
+    try {
+      execSync(`cd ../../../ && ./scripts/get-release-info.sh ${stageName}`);
```

Comment:
Does this command also change the directory for end user who ran the original `build-image` command? If yes, can we run `../../../scripts/get-release-info.sh` instead?

Reply:
`get-release-info.sh` assumes it is being called from the `service-workbench-on-aws` folder because inside it has paths for CHANGELOG.md and the stage file folder hardcoded. What is the case that someone would can the "original" `build-image` command? Is that different than `pnpx sls build-image`?